Assets/Scripts/Characters/Humans/Bullet.cs

```diff
@@ -504,6 +479,23 @@ private void Update()
                 }
             }
         }
-    }
-}
 
+        public static bool IsLeft(Source source) =>
+            source switch
+            {
+                Source.BeltLeft => true,
+                Source.GunLeft => true,
+                Source.BeltRight => false,
+                Source.GunRight => false,
+                _ => throw new ArgumentOutOfRangeException(nameof(source), source, null)
+            };
+
+        public enum Source
```

Comment:
Name suggestions are welcome.

Reply:
Perhaps name it something like HookSource would be better, as Source seems too generic